storage/innobase/include/srv0srv.h

```diff
@@ -78,7 +78,7 @@ struct MY_ALIGNED(CPU_LEVEL1_DCACHE_LINESIZE) simple_counter
 /** Global counters used inside InnoDB. */
 struct srv_stats_t
 {
-	typedef ib_counter_t<ulint, 64> ulint_ctr_64_t;
+	typedef ib_counter_t<ulint, IB_N_SLOTS> ulint_ctr_64_t;
```

Comment:
What is 64 in `ulint_ctr_64_t` now? Please, rename to something.

Reply:
right. Now changed to ulint_ctr_n_t.